aiida_quantumespresso/parsers/parse_xml/pw/versions.py

```diff
@@ -30,14 +30,19 @@ def get_xml_file_version(xml):
         raise XMLUnsupportedFormatError('unrecognized XML file version')
 
 
-def get_schema_filepath(xml):
+def get_schema_filepath(xml, schema):
```

Comment:
If we want to use this both for cp and pw, please move one level up in the folder structure (not inside `pw`). Otherwise, since the function is so simple you can just copy-paste the function twice for the pw and cp implementation, it's really just concatenating filenames.

Reply:
see 5fbc3ef5 major refactoring